spark/src/main/java/com/netflix/iceberg/spark/SparkExpressions.java

```diff
@@ -334,10 +336,19 @@ private static long dayToTimestampMicros(int daysFromEpoch) {
     }
   }
 
-  public static Expression convert(com.netflix.iceberg.expressions.Expression filter,
-                                   Schema schema) {
-    return visit(Binder.bind(schema.asStruct(), filter, true), new ExpressionToSpark(schema));
-  }
+// FIXME: Let's consider removing this method as no-one is using it.
```

Comment:
+1

Reply:
In fact, I think that the entire class can be removed. Spark removed Expression from the DSv2 API.

Reply:
Right, I took care of this separately via #94.